test/metabase/driver/mysql_test.clj

```diff
@@ -815,3 +819,18 @@
             (is (= (->> unbinned-query qp/process-query mt/cols (map :database_type)
                         (map #(get {"TIMESTAMP" "DATETIME"} % %)))
                    (->> binned-query   qp/process-query mt/cols (map :database_type))))))))))
+
+(deftest have-select-privelege?-timeout-test
```

Comment:
Can this be `:parallel` ?

Reply:
These are using a `with-redefs` to modify the generated select statement so i don't think they can be parallel.